ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -186,15 +187,53 @@ Item {
         textFormat: Text.RichText
     }
 
+    StyledText {
+        id: dateGroupLbl
+        font.pixelSize: 13
+        color: Style.current.darkGrey
+        horizontalAlignment: Text.AlignHCenter
+        anchors.horizontalCenter: parent.horizontalCenter
+        text: {
+            if (prevMessageIndex == -1) return ""; // identifier
+
+            let now = new Date()
+            let yesterday = new Date()
+            yesterday.setDate(now.getDate()-1)
+
+            let prevMsgTimestamp = chatsModel.messageList.getMessageData(prevMessageIndex, "timestamp")
+            var currentMsgDate = new Date(parseInt(timestamp, 10));
+            var prevMsgDate = prevMsgTimestamp === "" ? new Date(0) : new Date(parseInt(prevMsgTimestamp, 10));
+            if(currentMsgDate.getDay() != prevMsgDate.getDay()){
+                if (now.toDateString() == currentMsgDate.toDateString()) {
+                    return qsTr("Today")
+                } else if (yesterday.toDateString() == currentMsgDate.toDateString()) {
+                    return qsTrId("yesterday")
```

Comment:
When using `qsTrId`, it is better to add a comment above to say what is the source of the id. In this case:
```
//% "Yesterday"
return qsTrId("yesterday")
```